docs/release_notes/5.0.25-22.md

```diff
@@ -0,0 +1,30 @@
+# Percona Server for MongoDB 5.0.25-22 (2024-03-)
+
+[Installation](../install/index.md){.md-button}
+
+Percona Server for MongoDB 5.0.25-22 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for [MongoDB 5.0.25 Community Edition](https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.25---february-28--2024).
+
+It supports MongoDB 5.0.25 protocols and drivers.
+
+!!! warning
+
+    Due to [CVE-2024-1351](https://www.cve.org/CVERecord?id=CVE-2024-1351), in all MongoDB versions prior to 4.4.29, the `mongod` server allows incoming connections to skip peer certificate validation which results in untrusted connections to succeed. This issue occurs when the `mongod` is started with TLS enabled (`net.tls.mode` set to `allowTLS`, `preferTLS`, or `requireTLS`) and without a `net.tls.CAFile` configured. For details, see [SERVER-72839](https://jira.mongodb.org/browse/SERVER-72839).
+
+    The issue is fixed upstream in versions 4.4.29, 5.0.25, 6.0.14 and 7.0.6 and in Percona Server for MongoDB 4.4.29-28, 5.0.25-22, 6.0.14-11 and 7.0.6-4. Now, configuring MongoDB to use TLS requires specifying the value for the `--tlsCAFile` flag, the `net.tls.CAFile `configuration option, or the `tlsUseSystemCA` parameter.
+
+## Release Highlights
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
+
+* [SERVER-72839](https://jira.mongodb.org/browse/SERVER-72839) - Fixed the issue with missing peer certificate validation if neither CAFile nor clusterCAFile is provided.
+* [SERVER-83091](https://jira.mongodb.org/browse/SERVER-83091) - Extended the `collMod` command to check and fix invalid boolean index options.
+* [SERVER-82353 ](https://jira.mongodb.org/browse/SERVER-82353) - Fixed the issue with multi-document transactions missing documents when the movePrimary operation runs concurrently by detecting placement conflicts in multi-document transactions.
+* [SERVER-83564](https://jira.mongodb.org/browse/SERVER-83564) - Add an index on the process field for the `config.locks` collection to ensure update operations on it are completed even in heavy loaded deployments.
+* [WT-10017](https://jira.mongodb.org/browse/WT-10017) - Removed the unstable historical versions at the end of rollback to stable.
+
+Find the full list of changes in the [MongoDB 5.0.25 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.25---february-28--2024).
+
+## Bugs Fixed
+
+* [PSMDB-1434](https://perconadev.atlassian.net/browse/PSMDB-1434) - Fixed the auditing behavior not to log CRUD operations
```

Comment:
This title is not clear to me. Was there not enough CRUD operations logged or vice versa? (In fact we removed excessive logging).